packages/core/content-type-builder/server/src/services/architect.ts

```diff
@@ -31,11 +31,12 @@ const systemPrompt = `
             </schema_kind>
 
             <info_properties>
+              You MUST provide all the following properties:
+              - name: Same as pluralName
```

Comment:
Does it actually work? I've tried similar prompts but didn't manage to fix the issue